sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -89,6 +89,8 @@ static inline rom_error_t mask_rom_irq_error(void) {
  */
 static rom_error_t mask_rom_init(void) {
   CFI_FUNC_COUNTER_INCREMENT(rom_counters, kCfiRomInit, 1);
+  // Write OTP value to bits 0 to 5 of the cpuctrl CSR.
+  CSR_WRITE(CSR_REG_CPUCTRL, otp_read32(OTP_CTRL_PARAM_CREATOR_SW_CFG_CPUCTRL_OFFSET) & 0x3f);
```

Comment:
Since this enables some of the Ibex countermeasures, can you also add a couple CSR_READ checks later on to ensure the CSR has the expected value?

Reply:
Done